### DISCUSSION TITLE
License locked (revalidating)

### DISCUSSION THREAD

**Discussion by Tianyu-Yu** [aclap-dev/video-downloadhelper#333] [Help and Q&A]
### Your question

Hi my account says License locked (revalidating), how do I activate the converter licensing?

### Addon & CoApp version and details

Video DownloadHelper
Version 8.2.2.9 (stable)
Browser locale: en-US
Production build
Built on 2024-03-19
Build options: browser=chrome, linuxlic=false, noyt=true, target=google

**Comment by jcfrog** [comment]
Hi,
It's a security check every 8 installs.
You should have received an email with a link to unlock the license. Please check your spam folder.
If you don't find it, send us an email at licenses@downloadhelper.net
